newIDE/app/src/ObjectsList/EnumerateObjects.js

```diff
@@ -221,28 +222,74 @@ export const enumerateGroups = (
 export const enumerateObjectsAndGroups = (
   globalObjectsContainer: gdObjectsContainer,
   objectsContainer: gdObjectsContainer,
-  type: ?string = undefined
+  type: ?string = undefined,
+  behaviorConstraints?: Array<{
+    behaviorName: string,
+    behaviorType: string,
+  }> = []
 ) => {
   const filterObject = (object: gdObject): boolean => {
+    const behaviorNames = behaviorConstraints
```

Comment:
For clarity, could you rename it `objectBehaviorsNames?`

Reply:
I removed this part of the code.